src/endpoints/sources.js

```diff
@@ -391,6 +403,29 @@ async function sendAny(req, res, url, accept) {
 }
 
 
+/**
+ * Generate an Express endpoint that receives a group overview
+ * determined by the request.
+ *
+ * @param {sourceExtractor} sourceExtractor - Function to provide the Source instance from the request
+ * @returns {expressEndpointAsync}
+ */
+function receiveGroupOverview(sourceExtractor) {
+  return async (req, res) => {
+    const groupSource = sourceExtractor(req);
+
+    authz.assertAuthorized(req.user, authz.actions.Read, groupSource);
+
+    const urlCallback = async (method, headers) => await groupSource.urlFor("group-overview.md", method, headers);
+
+    return await receiveAny(req, res,
+      urlCallback,
+      "text/markdown" // TODO: put this in a class?
+    );
```

Comment:
Ditto about inlining `urlCallback` and not worrying about abstracting the media type.

Reply:
Inlined: 21a91cfd89eafae234ec860224638f94259416fb

Removed TODO: ed4ce9bb50750c03c5d3f1ffb801c13908795652